### PR TITLE
Onboard  devfile registry repository

### PR DESCRIPTION
**Pull request by josephca**
Onboard  devfile registry repository

### CODE REVIEW

File: ci-operator/config/devfile/registry/devfile-registry-main__v4.5.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    test:
+    - as: registry-integration-test-steps
+      commands: |
+        export PATH=$PATH:/tmp/bin
+        mkdir /tmp/bin
+        curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.5/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
+        chmod ug+x /tmp/bin/oc
+        .ci/openshift_integration.sh
+      dependencies:
+      - env: REGISTRY
```
[the remark this answers is not in the document]
Comment (josephca) [reply ci-operator/config/devfile/registry/devfile-registry-main__v4.5.yaml:37]: Thanks, John. You're right. I'll update the variable name.

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**